app/controllers/weather_controller.rb

```diff
@@ -11,5 +11,6 @@ def index
 			temp_f = parsed_json['current_observation']['temp_f']
 			puts "Current temperature in #{location} is: #{temp_f}\n"
 		end
+		#changes go in here
```

Comment:
here in the index method of the weather_controller you're creating the cariable temp_f which holds the temperature for Austin.  They way you pass variables to the view is by making them instance variables(@temp_f)